include/coro/io_scheduler.hpp

```diff
@@ -178,7 +178,7 @@ class io_scheduler : public std::enable_shared_from_this<io_scheduler>
      * @param task The task to execute on this io_scheduler.  It's lifetime ownership will be transferred
      *             to this io_scheduler.
      */
-    auto schedule(coro::task<void>&& task) -> void;
+    auto schedule(coro::task<void>&& task) -> bool;
```

Comment:
Suggestion: Also add to the documentation above what the return value means?

Reply:
Good idea, I'll also add docs for the `task_container::start` function as well.